docs/pages/identity-security/how-to-use.mdx

```diff
@@ -37,6 +41,14 @@ user or bot.
 
 ![Standing Privileges](../../img/access-graph/standing-privileges.png)
 
+## Dashboard: Identity Activity Center - Investigate
```

Comment:
Is the name of the heading the same as the name of the view? If not, I would tighten up the heading a bit, since it's on the long side. We also use sentence case for headings, so any common nouns would be lowercased.